### PR TITLE
Upgrade scala versions

### PR DESCRIPTION
**Pull request by maekl**
Upgrade to the latest Scala versions: 2.13.2, 2.12.11 and 2.12.11

This required a few other changes to be made

- tut replaced with mdoc
- upgrade sbt-microsites to latest version
- upgrade sbt-doctest to latest version

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,11 +15,11 @@ install:
 - gem update --system
```
Comment (zainab-ali) [inline .travis.yml:15]: We should upgrade ruby here too.  Could you use `2.6.5` as recommended in the [microsite docs](https://47degrees.github.io/sbt-microsites/docs/getting-started/)?

File: .travis.yml [hunk 2]
```diff
@@ -15,11 +15,11 @@ install:
...
 - gem install sass
 - gem install ruby_dep -v 1.3.1
-- gem install jekyll -v 3.2.1
+- gem install jekyll -v 4.0.0
 before_script:
 - if [ $TRAVIS_PULL_REQUEST = 'false' ]; then git checkout -qf $TRAVIS_BRANCH; fi
 script:
-- if [ $TRAVIS_SCALA_VERSION = '2.12.8' ];
+- if [ $TRAVIS_SCALA_VERSION = '2.12.11' ];
```
Comment (zainab-ali) [inline .travis.yml:22]: We should be able to build this on `2.13` now that we have mdoc

**Comment by zainab-ali** [comment]
Thanks @maekl !  The build for this is [failing in Travis](https://travis-ci.org/github/to-ithaca/libra/jobs/678969646) as a result of the jekyll upgrade.  Could you upgrade ruby too?

I'm running into scss path issues running locally.  There might be something else that needs to be configured for sbt-microsite.

**Comment by maekl** [comment]
Sure, this PR was apparently a bit a premature. I'll have another a look and fix all the issues

File: .travis.yml (hunk 1 above)
Comment (maekl) [reply .travis.yml:15]: Of course, fixed. I somehow missed the entire tut->mdoc migration

File: .travis.yml (hunk 2 above)
Comment (maekl) [reply .travis.yml]: Fixed

Comment (maekl) [reply .travis.yml]: Hmm, it is now failing for all versions except 2.13.2. Don't understand why yet.

**Pull request merged**